draft-ietf-capport-architecture.xml

```diff
@@ -869,6 +869,13 @@ o . . . . . . . . . . . . . . . . . . .| |. . . . . . . . . . . o
          credentials to an attacker. Browsers can authenticate servers but
          cannot detect cleverly misspelled domains, for example.
         </t>
+        <t>
+         Further, the possibility of an on-path attacker in an attacking network
+         introduces some risks. The attacker could redirect traffic to arbitrary
+         destinations with malicious payloads. The attacker could analyze the user's
```

Comment:
```suggestion
         destinations. The attacker could analyze the user's
```

No point in going beyond the attack.  Malicious payloads is something we should be well-prepared for (even if in practice we aren't).

Reply:
Thanks